samples/files.js

```diff
@@ -154,6 +154,79 @@ async function uploadFileWithKmsKey(bucketName, filename, kmsKeyName) {
   // [END storage_upload_with_kms_key]
 }
 
```

Comment:
Can you add `sample-metadata` to this sample?

Here's an example: #793

Reply:
Done
Thanks